ios/ReactNativePageView.m

```diff
@@ -390,6 +382,16 @@ - (void)scrollViewDidScroll:(UIScrollView *)scrollView {
         absoluteOffset =  1 - absoluteOffset;
     }
     
+    if (!_overdrag) {
```

Comment:
Could you update your branch with the latest master ? 
https://github.com/callstack/react-native-pager-view/pull/363/files

Reply:
Done!